src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -565,13 +600,21 @@ export class ExchangeWrapper extends ContractWrapper {
         return eventEmitter;
     }
     /**
-     * Returns the ethereum address of the current exchange contract
+     * Returns the ethereum addresses of all available exchange contracts
      * on the network that the provided web3 instance is connected to
-     * @return  The ethereum address of the current exchange contract.
+     * @return  The ethereum addresses of all available exchange contract.
      */
-    public async getContractAddressAsync(): Promise<string> {
-       const exchangeContract = await this._getExchangeContractAsync();
-       return exchangeContract.address;
+    public async getAvailableContractAddressedAsync(): Promise<string[]> {
```

Comment:
Let's also add a method that returns the Exchange contracts that are currently authorized on the `Proxy` contract of a given network. It's not very useful to retrieve an exchange contract that isn't operational anymore. Maybe we could call this `getProxyAuthorizedContractAddressesAsync(networkId: number)`

*Unrelated*: This is a type to fix here: `getAvailableContractAddressedAsync` should be `getAvailableContractAddressesAsync`